### PR TITLE
AT_18.02.001 | <Profile Page>  Verify that the user can edit the status description

### PR DESCRIPTION
**Pull request by AleksandrAkimovChg**
https://trello.com/c/Rf4aGtqj

### CODE REVIEW

**Review by IrinaLeibutina — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

File: cypress/e2e/editStatusDescription.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+describe('editStatusDescription', () => {
+
+    it('AT_18.02.001 | Verify that the user can edit the status description', () => {
+        cy.get('a[href="/user/admin"]').click();
+        cy.location('pathname').should('eq', '/user/admin/');
+        cy.get('#description-link.jenkins-button').click();
+        cy.get('textarea.jenkins-input').clear();
+        cy.get('textarea.jenkins-input').type('Hello world!');
```
Comment (Rom-T) [inline cypress/e2e/editStatusDescription.cy.js:10]: Please move `Hello world!` in the fixtures

File: cypress/e2e/editStatusDescription.cy.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        cy.get('button[name=Submit]').click();
+        cy.get('#description').should('contain', "Hello world!");
```
Comment (Rom-T) [inline cypress/e2e/editStatusDescription.cy.js:12]: Please move `Hello world!` in the fixtures

File: cypress/e2e/editStatusDescription.cy.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+describe('editStatusDescription', () => {
+
+    it('AT_18.02.001 | Verify that the user can edit the status description', () => {
+        cy.get('a[href="/user/admin"]').click();
+        cy.location('pathname').should('eq', '/user/admin/');
```
Comment (Rom-T) [inline cypress/e2e/editStatusDescription.cy.js:7]: please rename data `'/user/admin/'` to `/user/${Cypress.env('local.admin.username').toLowerCase()}/`

File: cypress/e2e/editStatusDescription.cy.js (hunk 1 above)
Comment (AleksandrAkimovChg) [reply cypress/e2e/editStatusDescription.cy.js]: fix

File: cypress/e2e/editStatusDescription.cy.js (hunk 2 above)
Comment (AleksandrAkimovChg) [reply cypress/e2e/editStatusDescription.cy.js]: fix

File: cypress/e2e/editStatusDescription.cy.js (hunk 3 above)
Comment (AleksandrAkimovChg) [reply cypress/e2e/editStatusDescription.cy.js]: fix

**Comment by AleksandrAkimovChg** [comment]
@Rom-T please, resolve

**Pull request merged**